x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/boosts/boosts.scss

```diff
@@ -0,0 +1,32 @@
+.boosts {
```

Comment:
[optional] Same comment here about nesting `.boosts .boosts__child` vs just `.boosts__child` - it also does feel a little odd we're not putting anything on `.boosts` specifically in that scenario 🤔 I'd be fine with either one of these options:

```scss
.boosts__select {
  // etc
}

.boosts__item {
 // etc
}
```

or

```scss
.boostSelect {
}

.boostItem {
}
  .boostItem__content {
  }
```

Reply:
https://github.com/elastic/kibana/pull/90621/commits/e7f6bf47eee03054c4d73e7ccb27e958e65685b6